sled-agent/src/http_entrypoints.rs

```diff
@@ -25,7 +25,7 @@ use uuid::Uuid;
 
 use super::sled_agent::SledAgent;
 
-use propolis_client::api::VolumeConstructionRequest;
+use crucible_client_types::VolumeConstructionRequest;
```

Comment:
Will the generated client not re-export types it uses?

Reply:
kind of a rough edge i think: progenitor re-declares a copy of them based on the openapi spec it's given (which is of course generated with the `derive(JsonSchema)` on the original `crucible_client_types`).  unless there's a fancy feature of progenitor i'm missing, this means boilerplate to convert between the two

Reply:
ah right i know you added the From impls for the `generated-migration` feature but I wonder if its worth just having in general too

Reply:
i get the distinct feeling it might be, yeah.